arelle/plugin/inlineXbrlDocumentSet.py

```diff
@@ -514,8 +515,20 @@ def commandLineOptionExtender(parser, *args, **kwargs):
                       help=_("Encode saved xml characters (&#x80; and above)"))
     parser.add_option("--encodesavedxmlchars",  # for WEB SERVICE use
                       action="store_true",
-                      dest="encodeSavedXmlChars",
+                      dest="defaultNamespace",
                       help=SUPPRESS_HELP)
+    parser.add_option("--defaultNamespace",
+                      action="store",
+                      default='',
+                      dest="defaultNamespace",
+                      help=_("The namespace to use for the base namespace, defaults to the default namespace"),
```

Comment:
`xbrliNamespacePrefix` and use `None` as the default value.
```suggestion
    parser.add_option("--xbrliNamespacePrefix",
                      action="store",
                      dest="xbrliNamespacePrefix",
                      help=_("The namespace prefix to use for http://www.xbrl.org/2003/instance. It's used as the default namespace when unset."),
```